docs/articles/scaleout.md

```diff
@@ -122,22 +179,23 @@ From zot release v2.1.9 and newer, the `sessionDriver` configuration can also be
     "tls": {
       "cacert": "test/data/ca.crt"
     }  
+  },
+  "extensions": {
```

Comment:
Since `ui` depends on `search`, `search` should also be enabled. Let's add it here.